lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -337,6 +337,7 @@ defmodule ExUnit.Case do
       end
 
   """
+  @doc since: "1.0.4"
   defmacro test(message, var \\ quote(do: _), contents) do
```

Comment:
I see it introduced in `1.0.4` but this signature match in `1.1.0`. So I'm not sure what version should be. 🙇